salvo/README.md

```diff
@@ -78,10 +122,19 @@ source:
   source_url: https://github.com/envoyproxy/nighthawk.git
 ```
 
-In this example, the v1.16.0 Envoy tag is pulled and an Envoy image generated where the Envoy binary has profiling enabled.  The user may specify option strings supported by bazel to adjust the compilation process.
+`scavengingBenchmark`: It will run scavenging benchmarks.
+
+`source.identity`: Specify whether this source location is Envoy or NightHawk.
+
+`source.commit_hash`: Specify a commit hash if applicable. If not specified we will determine this from the source tree. We will also use this field to identify the corresponding NightHawk or Envoy image used for the benchmark.
```

Comment:
(nit) We seem to have a couple of ways how we spell Nighthawk in this document. Would it be worthwhile to unify them and use just one format?

Reply:
Thanks. I've unified them to `Nighthawk`.